src/main/resources/db/migration/V1.3__list_type_subscription.sql

```diff
@@ -0,0 +1,27 @@
+--
```

Comment:
We will need a DDL query to create the subscription_list_type table first, as in higher environments only flyway creates / modifies tables

Reply:
done